docs/en/Running-Tachyon-on-EC2.md

```diff
@@ -113,7 +128,13 @@ Default port for Hadoop Web UI is **50070**.
 Visit `http://{MASTER_IP}:{PORT}` in the browser to access the Web UIs.
 
 You can also monitor the instances state through
-[AWS web console](https://console.aws.amazon.com/console/home?region=us-east-1).
+[AWS web console](https://console.aws.amazon.com/console).
+Make sure that you are in the console for the region where you started the cluster.
+
+Here are some scenarios when you may want to check the console:
+ - When the cluster creation fails, check EC2 instances status/logs.
+ - After the cluster is destroyed, confirm EC2 instances are terminated.
+ - When you no longer need the cluster, make sure EC2 instances and S3 buckets are NOT costing you extra money.
```

Comment:
But as the default UNDERFS with EC2 has been changed to S3, would it make sense to just double check S3 bucket deleted as expected?